__tests__/app.test.js

```diff
@@ -336,6 +336,31 @@ describe("/api/articles", () => {
       return request(app)
         .post("/api/articles/1/comments")
         .send(testComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("not found");
+        });
+    });
+    it("404: username not found", () => {
+      const testComment = {
+        username: "wrong_username",
+        body: "test body",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(testComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("not found");
+        });
+    });
+    it("400: missing required field/key", () => {
+      const testComment = {
+        body: "test body",
+      };
+      return request(app)
+        .post("/api/articles/20/comments")
+        .send(testComment)
         .expect(400)
         .then(({ body }) => {
           expect(body.msg).toBe("bad request");
```

Comment:
`it("400: article_id does not exist", () => {
      const testComment = {
        username: "butter_bridge",
        body: "test body",
      };`
      
      Should be 404 it doesnt exist/is not found in the db to post to.